src/app/services/dirty-dialog.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DirtyDialogService } from './dirty-dialog.service';
+
+describe('DirtyDialogService', () => {
+  let service: DirtyDialogService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(DirtyDialogService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
```

Comment:
Is this useful? @the-greenman

Reply:
I'm wondering about these auto generated stubs. They do provide an essential bit of test coverage. They do actually make sure that the object gets created. 

In this case I don't actually know how to test the current functionality of this service outside of Cypress, but functionality may evolve. 

Getting back to the Angular default standard of always having a spec test for a file may be a good objective.